<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ASCSBCWTL016" ignored="0" name="RestTest" tests="24" failures="5" timestamp="05 Jun 2020 08:01:08 GMT" time="21.627" errors="0">
  <testcase name="validateContentType" time="0.008" classname="com.api.qa.test.TC002_GET_Single_Employee"/>
  <testcase name="validateResponseBody" time="0.001" classname="com.api.qa.test.TC002_GET_Single_Employee">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.api.qa.test.TC002_GET_Single_Employee.validateResponseBody(TC002_GET_Single_Employee.java:35)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateResponseBody -->
  <testcase name="validateResponseTime" time="0.001" classname="com.api.qa.test.TC002_GET_Single_Employee"/>
  <testcase name="validateStatusCode" time="0.001" classname="com.api.qa.test.TC002_GET_Single_Employee">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at com.api.qa.test.TC002_GET_Single_Employee.validateStatusCode(TC002_GET_Single_Employee.java:43)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateStatusCode -->
  <testcase name="validateStatusLine" time="0.001" classname="com.api.qa.test.TC002_GET_Single_Employee">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 400 Bad Request]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 400 Bad Request]
at com.api.qa.test.TC002_GET_Single_Employee.validateStatusLine(TC002_GET_Single_Employee.java:63)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateStatusLine -->
  <testcase name="validateContentType" time="0.0" classname="com.api.qa.test.TC005_Delete_Employee"/>
  <testcase name="validateResponseBody" time="0.001" classname="com.api.qa.test.TC005_Delete_Employee">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.api.qa.test.TC005_Delete_Employee.validateResponseBody(TC005_Delete_Employee.java:42)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateResponseBody -->
  <testcase name="validateStatusCode" time="0.001" classname="com.api.qa.test.TC005_Delete_Employee"/>
  <testcase name="validateStatusLine" time="0.001" classname="com.api.qa.test.TC005_Delete_Employee"/>
  <testcase name="validateContentType" time="0.001" classname="com.api.qa.test.TC003_POST_Create_Employee"/>
  <testcase name="validateResponseBody" time="0.002" classname="com.api.qa.test.TC003_POST_Create_Employee"/>
  <testcase name="validateResponseTime" time="0.002" classname="com.api.qa.test.TC003_POST_Create_Employee"/>
  <testcase name="validateStatusCode" time="0.002" classname="com.api.qa.test.TC003_POST_Create_Employee"/>
  <testcase name="validateStatusLine" time="0.001" classname="com.api.qa.test.TC003_POST_Create_Employee"/>
  <testcase name="validateContentType" time="0.001" classname="com.api.qa.test.TC001_Get_All_Employees"/>
  <testcase name="validateResponseBody" time="0.0" classname="com.api.qa.test.TC001_Get_All_Employees"/>
  <testcase name="validateResponseTime" time="0.001" classname="com.api.qa.test.TC001_Get_All_Employees"/>
  <testcase name="validateStatusCode" time="0.001" classname="com.api.qa.test.TC001_Get_All_Employees"/>
  <testcase name="validateStatusLine" time="0.001" classname="com.api.qa.test.TC001_Get_All_Employees"/>
  <testcase name="validateContentType" time="0.001" classname="com.api.qa.test.TC004_PUT_Update_Employee"/>
  <testcase name="validateResponseBody" time="0.001" classname="com.api.qa.test.TC004_PUT_Update_Employee">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.api.qa.test.TC004_PUT_Update_Employee.validateResponseBody(TC004_PUT_Update_Employee.java:52)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateResponseBody -->
  <testcase name="validateResponseTime" time="0.001" classname="com.api.qa.test.TC004_PUT_Update_Employee"/>
  <testcase name="validateStatusCode" time="0.0" classname="com.api.qa.test.TC004_PUT_Update_Employee"/>
  <testcase name="validateStatusLine" time="0.001" classname="com.api.qa.test.TC004_PUT_Update_Employee"/>
</testsuite> <!-- RestTest -->
